lib/alerts_viewer_web/live/alerts_to_close_live.ex

```diff
@@ -130,15 +130,20 @@ defmodule AlertsViewerWeb.AlertsToCloseLive do
 
     duration = DateTime.diff(current_time, alert.created_at, :minute)
 
-    peak =
+    headways =
       route_ids
       |> Enum.map(fn route_id ->
         stats_by_route
         |> RouteStats.max_headway_deviation(route_id)
         |> DateTimeHelpers.seconds_to_minutes()
       end)
       |> Enum.reject(&is_nil/1)
-      |> Enum.max()
+
+    peak =
+      case(length(headways) > 0) do
```

Comment:
suggestion (non-blocking): I think you could also probably do `case headways` and then have the first condition be `[]`. I suspect that would probably be more performant (constant time vs O(n)), but I also don't think it realistically matters in practice.